clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -235,17 +236,17 @@ def _get_pods_info(
             "exclude_daemonset_pods",
             default=staticconf.read_bool("exclude_daemonset_pods", default=False),
         )
-        all_pods = self._core_api.list_pod_for_all_namespaces().items
-        for pod in all_pods:
-            if self._pod_belongs_to_pool(pod):
```

Comment:
Look we don't need _pod_belongs_to_pool method.

Reply:
Oh yes, changed it!